com.unity.netcode.gameobjects/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -388,6 +388,13 @@ internal NetworkObject CreateLocalNetworkObject(NetworkObject.SceneObject sceneO
                         NetworkLog.LogError($"{nameof(NetworkPrefab)} hash was not found! In-Scene placed {nameof(NetworkObject)} soft synchronization failure for Hash: {globalObjectIdHash}!");
                     }
                 }
+
+                // Since this NetworkObject is an in-scene placed NetworkObject, if it is disabled then enable it so
+                // NetworkBehaviours will have their OnNetworkSpawn method invoked
+                if (networkObject != null && !networkObject.gameObject.activeInHierarchy)
+                {
+                    networkObject.gameObject.SetActive(true);
```

Comment:
Is this really the behavior we want here? Doesn't this effectively mean users can't disable scene objects at all because we're just going to undo it? Maybe I'm missing context though.

Reply:
Look at the comment I left for Jeffrey above, he had the same question.
The short answer is:
This occurs only on the client-side when the server has sent a message to the client to spawn the the disabled NetworkObject.  So, if a user disables a NetworkObject and their code never attempts to spawn it then it will remain disabled.

Reply:
K, sounds fine to me.